pkgs/development/libraries/gettext/default.nix

```diff
@@ -28,6 +28,12 @@ stdenv.mkDerivation (rec {
         "gt_cv_func_CFLocaleCopyCurrent=no"
       ]);
 
+  patchPhase = ''
+   substituteInPlace gettext-tools/projects/KDE/trigger --replace "/bin/pwd" pwd
+   substituteInPlace gettext-tools/projects/KDE/trigger --replace "/bin/pwd" pwd
+   substituteInPlace gettext-tools/src/project-id --replace "/bin/pwd" pwd
```

Comment:
I see &ndash; it's an indirect build-time-only dependency. I think we can go with this PR as it is then. It would be more complicated to get this right, as our bootstrapping seems much about hand-crafting.